watchface

```diff
@@ -13,20 +13,21 @@ Utility functions for AsteroidOS watchfaces.  By default, uses "Developer Mode"
 over ssh, but can also use "ADB Mode" using ADB.
 
 Available options:
--h or --help    prints this help screen and quits
--a or --adb     uses ADB command to communicate with watch
+-h or --help    print this help screen and quits
+-a or --adb     use ADB command to communicate with watch
 -b or --boot    reboot watch after deploying multiple watchfaces
 -g or --gui     use the GTK+ gui
--p or --port    specifies a port to use for ssh and scp commands
--r or --remote  specifies the remote (watch)  name or address for ssh and scp commands
--q or --qemu    communicates with QEMU emulated watch (same as -r localhost -p 2222 )
--w or --wall WP sets the wallpaper for deploy or test to the named file
+-p or --port    specify an IP port to use for ssh and scp commands
+-r or --remote  specify the remote (watch)  name or address for ssh and scp commands
+-q or --qemu    communicate with QEMU emulated watch (same as -r localhost -p 2222 )
+-w or --wall WP set the wallpaper for deploy or test to the named file
 
 Available commands:
-version         displays the version of this program and exits
-deploy WF       pushes the named watchface to the watch and activates it
-clone WF NEWWF  clones the named watchface WF to new watchface NEWWF
-test WF         tests the named watchface on the computer using qmlscene
+update          use git to update your local copy of the unoffical-watchfaces repository
+version         display the version of this program and exits
```

Comment:
`.. this program and exits` should be `exit` now
same for `activates` to `activate` in next line
and in line 16
`quits` to `quit`

Reply:
You're right.  Thanks!